lib/expr.c

```diff
@@ -5029,14 +5029,17 @@ grn_table_select(grn_ctx *ctx, grn_obj *table, grn_obj *expr,
             data.min_id = GRN_ID_NIL;
           }
         }
-        GRN_QUERY_LOG(ctx, GRN_QUERY_LOG_SIZE,
-                      ":", "filter(%d)%s",
-                      grn_table_size(ctx, data.res),
-                      grn_table_select_inspect_condition(
-                        ctx,
-                        &condition_inspect_buffer,
-                        si,
-                        e));
+        const char *condition = grn_table_select_inspect_condition(ctx,
+                                                                   &condition_inspect_buffer,
+                                                                   si,
+                                                                   e);
+        if (grn_strncasecmp(condition, ": between", 9) != 0 ||
+            (grn_strncasecmp(condition, ": between", 9) == 0 && grn_table_size(ctx, data.res) > 0)) {
```

Comment:
Umm...
I think `filter(0)` is also important information.
However, I have no good idea against this modify.

Could you directly ask kou to review, please?

Reply:
@komainu8 
Ok, I'll do so.  Thank you for review!